### PR TITLE
Adding gluppi

### PR DESCRIPTION
**Pull request by leriano7**

### CODE REVIEW

File: apps/gluppi/gluppi.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+category: Social Networking
+keywords:
+    - telephony
+    - videoconference
+    - pbx
+    - chat
+    - video
+    - cloud
+    - storage
+license: Commertial
```
Comment (ckerr) [inline apps/gluppi/gluppi.yml]: Did you mean 'Commercial'?

Comment (leriano7) [reply apps/gluppi/gluppi.yml]: Hello! Yes, I meant commercial.

**Comment by leriano7** [comment]
I add the next image. We can see dev tools for this project.
image
Is that enough?
Thank you!

**Comment by leriano7** [comment]
You can also download a dev tools version here: https://gluppi.com/apps/v1.2.7-desarrollo/setup-64.exe
Thank you again!

File: apps/gluppi/gluppi.yml (hunk 1 above)
Comment (leriano7) [reply apps/gluppi/gluppi.yml]: Fixed in last commit

**Pull request merged**